### PR TITLE
deps: bump ibc-go v7.0.0 rc1

### PR DESCRIPTION
**Pull request by crodriguezvega**

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -764,14 +764,14 @@ func GetMaccPerms() map[string][]string {
 func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino, key, tkey storetypes.StoreKey) paramskeeper.Keeper {
 	paramsKeeper := paramskeeper.NewKeeper(appCodec, legacyAmino, key, tkey)
 
-	paramsKeeper.Subspace(authtypes.ModuleName)
-	paramsKeeper.Subspace(banktypes.ModuleName)
-	paramsKeeper.Subspace(stakingtypes.ModuleName)
-	paramsKeeper.Subspace(minttypes.ModuleName)
-	paramsKeeper.Subspace(distrtypes.ModuleName)
-	paramsKeeper.Subspace(slashingtypes.ModuleName)
+	paramsKeeper.Subspace(authtypes.ModuleName).WithKeyTable(authtypes.ParamKeyTable())
```
Comment (damiannolan) [inline app/app.go:767]: You could remove the subspace initialisation for all the sdk modules. Params are managed by them modules internally now so its not required, but fine by me if you want to leave it.
We only need `WithKeyTable(...)` here to perform upgrades from previous sdk versions to 0.47 in order to do the params migrations.

File: Makefile [hunk 2]
```diff
@@ -56,17 +56,17 @@ proto-lint:
 proto-check-breaking:
 	@$(DOCKER_BUF) breaking --against $(HTTPS_GIT)#branch=main
 
-TM_URL              = https://raw.githubusercontent.com/tendermint/tendermint/v0.34.0-rc6/proto/tendermint
+CMT_URL             = https://raw.githubusercontent.com/cometbft/cometbft/v0.37.0/proto/tendermint
 GOGO_PROTO_URL      = https://raw.githubusercontent.com/regen-network/protobuf/cosmos
-CONFIO_URL          = https://raw.githubusercontent.com/confio/ics23/v0.6.3
-SDK_PROTO_URL 		= https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.41.0/proto/cosmos
+CONFIO_URL          = https://raw.githubusercontent.com/cosmos/ics23/v0.7.1
```
Comment (damiannolan) [inline Makefile:61]: I guess this ain't really a confio url anymore 😅

**Review by damiannolan — APPROVED** [review]
All of the proto builds working fine? 

Noticed this repo still has a third_party directory.

**Comment by crodriguezvega** [comment]
> All of the proto builds working fine?

Yes, I run `make promo-all` and seemed to work fine.

**Comment by damiannolan** [comment]
@crodriguezvega I pushed a commit to attempt to fix the golangci-lint failures.

**Pull request merged**